ateam_vision_filter/src/types/robot_measurement.hpp

```diff
@@ -27,6 +27,34 @@ struct RobotMeasurement
 {
   Eigen::Vector2d position;
   double theta;
+
+  void invert()
+  {
+    position *= -1.0;
+    theta = angle_wrap_pi(theta + M_PI);
+  }
+
+  // TODO(Cavidano) MOVE THIS CONVERSION TO A COMMON PLACE
+  double angle_wrap_2pi(double angle_rad)
+  {
+    double pi2 = 2 * M_PI;
+    if (angle_rad >= pi2) {
+      angle_rad -= pi2 * std::floor(angle_rad / pi2);
+    }
+
+    if (angle_rad < 0) {
+      angle_rad += pi2 * std::ceil(std::abs(angle_rad) / pi2);
+    }
+    return angle_rad;
+  }
+
+  double angle_wrap_pi(double angle_rad)
+  {
+    if (angle_rad < -M_PI || M_PI < angle_rad) {
+      return angle_wrap_2pi(angle_rad + M_PI) - M_PI;
+    }
+    return angle_rad;
+  }

```

Comment:
These functions are already available in ROS's angles package.
See [`angles::normalize_angle_positive()`](http://docs.ros.org/en/humble/p/angles/generated/function_angles_8h_1a100ca493044236d7e099e08e78aab5bf.html) and [`angles::normalize_angle()`](http://docs.ros.org/en/humble/p/angles/generated/function_angles_8h_1a90ae8baf394c3ce775a3a14e48c7e7be.html).

Reply:
Sorry im just really used to my stupid new companies minimal depedencies rule. Its been rough....